docs/docs/50-user-guide/20-how-to-guides/40-working-with-stages.md

```diff
@@ -98,6 +106,30 @@ spec:
   # ...
 ```
 
+In the next example, the `prod` `Stage` requests `Freight` that has originated
+from the `my-warehouse` `Warehouse`, but indicates that it will accept such
+`Freight` only after it has been _verified_ in both the `test` and `uat`
+`Stage`s:
+
+```yaml
+apiVersion: kargo.akuity.io/v1alpha1
+kind: Stage
+metadata:
+   name: uat
+   namespace: kargo-demo
+spec:
+  requestedFreight:
+  - origin:
+      kind: Warehouse
+      name: my-warehouse
+    sources:
+      stages:
+      - test
+      - uat
+    availabilityStrategy: All
```

Comment:
Nit: If someone is really paying close attention, I think the previous example establishes uat as being downstream from test. That context makes this second example somewhat odd (albeit not invalid) as the pipeline would look like this:

```
test --> uat --> prod
  \___________/
```

With an `All` strategy, that isn't any different from:

```
test --> uat --> prod
```

If we just changed `test` to `qa` or something here, it might be less odd.